test/benchmarks/unitBench/list.bench.js

```diff
@@ -0,0 +1,109 @@
+/* eslint-disable no-restricted-modules */
+const chalk = require('chalk');
+const { List } = require('../../../lib/utils');
+const { createHistogram } = require('perf_hooks');
+
+const iterations = 100;
+const defaultItemsSize = 100000;
+const makeBigArray = (length = defaultItemsSize) => Array.from({ length }).fill(1);
+const makeReadableTime = nanoseconds => (nanoseconds / 1e6).toFixed(3).padStart(7, ' ');
+const printHistogram = (name, h) => {
+  console.log();
+  console.log(chalk.green(name));
+  console.log('-'.repeat(155));
+  process.stdout.write(`|  ${chalk.cyan('max')}:    ${chalk.red(makeReadableTime(h.max))} ms |`);
+  process.stdout.write(`  ${chalk.cyan('min')}:    ${chalk.red(makeReadableTime(h.min))} ms |`);
+  process.stdout.write(`  ${chalk.cyan('mean')}:   ${chalk.red(makeReadableTime(h.mean))} ms |`);
+  process.stdout.write(`  ${chalk.cyan('stddev')}: ${chalk.red(makeReadableTime(h.stddev))} ms |`);
+  process.stdout.write(
+    `  ${chalk.cyan('p90th')}:  ${chalk.red(makeReadableTime(h.percentile(90)))} ms |`
+  );
+  process.stdout.write(
+    `  ${chalk.cyan('p95th')}:  ${chalk.red(makeReadableTime(h.percentile(95)))} ms |`
+  );
+  process.stdout.write(
+    `  ${chalk.cyan('p99th')}:  ${chalk.red(makeReadableTime(h.percentile(99)))} ms |`
+  );
+  console.log('\n' + '-'.repeat(155));
+};
+
+void (function testArrayShift() {
```

Comment:
Any reason for the IIFEs?  Why not just throw them inside a `main` and call the `main` function?

Reply:
Added a main